src/vm/eventpipesession.cpp

```diff
@@ -319,7 +338,8 @@ void EventPipeSession::WriteEvent(EventPipeEventInstance &instance)
 
     if (m_pFile == nullptr)
         return;
-    m_pFile->WriteEvent(instance);
+    EventPipeSequencePoint sequencePoint;
+    m_pFile->WriteSequencePoint(&sequencePoint);
```

Comment:
nit: Use reference, or add PRECONDITION and if (pSequencePoint == nullptr) blocks.

Reply:
I assume you mean add PRECONDITION inside EventPipeFile::WriteSequencePoint... Done : )
I didn't add if(pSequencePoint == nullptr) because that is a runtime bug if it ever happens and I'd like to get the AV so we find and fix it quickly.